### DISCUSSION TITLE
Using different 404 pages for translations

### DISCUSSION THREAD

**Discussion by patiowebdesign** [kunalnagar/custom-404-pro#71] [Q&A]
Hello! This plugin did what I wanted which was create a specific page to be the 404 error and it worked fine for one language only.
But I am now designing a multilingual website (Polylang for the translations) and it's not recognising the translatated version of the 404 page, nor does it let me pick a different page for each language. Is there a way around this so that people you are navigating the site in language #2, will see the respective 404 page in their language as well instead of just the site's default?
Thank you, Sofia

**Comment by kunalnagar** [comment]
@patiowebdesign - thanks for the info - do you mind posting a few screenshots of the `Mode` options you see on the `Global Redirect` tab for the main site and the translated site?

This plugin does not support Multisite yet, unfortunately. I can investigate when I have some time 🙂
